abi/type.go

```diff
@@ -212,11 +214,15 @@ func parseTupleContent(str string) ([]string, error) {
 	var stack []int
```

Comment:
line 203 above, error should be "parsing error: tuple content should not start or end with comma"
also typo in comment noe = not

Reply:
👍 good find, will change accordingly.